src/sentry/models/deploy.py

```diff
@@ -23,6 +23,7 @@ class Deploy(Model):
     date_started = models.DateTimeField(null=True, blank=True)
     name = models.CharField(max_length=64, null=True, blank=True)
     url = models.URLField(null=True, blank=True)
+    notified = models.BooleanField(default=True, db_index=True)
```

Comment:
We still shouldn't do this just for sake of bad practice. You're right that it'd likely be ok in practice, but it'll be easier to just let it be `None`, and handle the case of `None` as `True`. We can just easily do a simple db migration to backfill all the values later then remove the code to handle None.

Could even inverse the logic and say `needs_notified` which might be a bit more natural feeling since `None` is then considered falsey.